docs/detailed-docs.md

```diff
@@ -1802,8 +1802,8 @@ Please refer to [Deploying Using GitHub App Authentication](#deploying-using-git
 
 # Troubleshooting
 
-See [troubleshooting guide](TROUBLESHOOTING.md) for solutions to various problems people have run into consistently.
+See [troubleshooting guide](https://github.com/actions-runner-controller/actions-runner-controller/blob/master/TROUBLESHOOTING.md) for solutions to various problems people have run into consistently.
```

Comment:
```suggestion
See [troubleshooting guide](/TROUBLESHOOTING.md) for solutions to various problems people have run into consistently.
```


I guess this might also work and be a little more fork friendly!

cc/ @Vijay-train Shall we also move the troubleshooting guide to a file under `docs`, eventually?

Reply:
Let me give this a shot